telegram/ext/filters.py

```diff
@@ -1020,6 +1020,14 @@ def filter(self, update):
 
         channel_posts = _ChannelPosts()
 
+        class _Poll(BaseFilter):
+            update_filter = True
+
+            def filter(self, update):
+                return update.message.poll
+
+        poll = _Poll()
+
```

Comment:
If I understood correctly, we wanted to remove this, didn't we?

Reply:
No, this is for the message attribute poll. So a Filter.poll, used inside of a message handler, would filter out messages with message.poll.

Reply:
Then whats the difference to #1673?

Reply:
Oh god I forgot about this PR